ckanext/canada/plugins.py

```diff
@@ -160,6 +160,7 @@ def get_helpers(self):
             'parse_release_date_facet',
             'is_ready_to_publish',
             'get_datapreview_recombinant',
+            'googleanalytics_id',
```

Comment:
this is already in the public plugin which we use on both portal and registry, so I don't think we need it here

Reply:
True. I just removed it